### DISCUSSION TITLE
Update group only after another package has been updated

### DISCUSSION THREAD

**Discussion by 1oglop1** [renovatebot/renovate#25555] [Request Help] [answered]
### How are you running Renovate?

self-hosted gitlab

### If you're self-hosting Renovate, tell us what version of Renovate you run.

N/A

### If you're self-hosting Renovate, select which platform you are using.

Gitlab

### Wanted end result.

Hello is it possible to update package group (or packages) only if the group contains the `core` package or if that package has been updated?

e.g.
Initial state 
```
core 1.0
pkgA 1.0
pkgB 1.0
```
Iteration 1
pkgA received a new version
```
pkgA 1.0 -> pkgA 1.1
```
At this point, I should not get PR. 

Iteration 2
core received a new version
```
core 1.0 -> 1.1
```
I'd like to get a PR like this
```
core 1.0 -> 1.1
pkgA 1.0 -> pkgA 1.1
```

### What you tried so far.

- looking at dependecy dashboard

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
Not possible
